notebooks/equivalence_equiangular_sampling.ipynb

```diff
@@ -0,0 +1,375 @@
+   "metadata": {},
+   "source": [
+    "## Equivalence of MW and Fejer type 1 sampling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "8559d609-67d5-4561-bb43-2de124f49e25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thetas_mw = s2fft.sampling.s2_samples.thetas(L, sampling=\"mw\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "bed1f19f-d015-4284-ade6-eeb8954d34b9",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.44879895, 1.34639685, 2.24399475, 3.14159265])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thetas_mw"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "ad22a641-3940-489e-98f5-315f0ca4c5e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n = L - 1/2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "de96933f-6b89-4796-bdcd-1eb4959a1a55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thetas_mw_fejer1 = (np.arange(0, n) + 0.5) * np.pi / n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "974e4e4d-0753-45da-8eab-ecf0cf21f501",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.44879895, 1.34639685, 2.24399475, 3.14159265])"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thetas_mw_fejer1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "01fd0a9d-180d-4275-b4bb-4e269aa1d466",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert np.allclose(thetas_mw, thetas_mw_fejer1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f13b08a-3df9-4999-a1a0-f022b36c7aa8",
+   "metadata": {},
+   "source": [
+    "## Equivalence of MWSS and Clenshaw-Curtis (Fejer type 2 with end-points) sampling "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "3df7cacf-90c1-44c5-9d84-4c0164b5df4e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.        , 0.78539816, 1.57079633, 2.35619449, 3.14159265])"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thetas_mwss = s2fft.sampling.s2_samples.thetas(L, sampling=\"mwss\")\n",
+    "thetas_mwss"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "c7e7af1f-d1dd-4d32-bbaf-3bf61f6026b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n = L"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "5334dba6-f247-45ea-8340-9e13d1321ab9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thetas_mwss_fejer2 = np.arange(1, n) * np.pi / n"
```

Comment:
Again without context unclear where this formula comes from / what is being shown here.